pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.poetry]
+name = "witty-wyverns"
+version = "0.1.0"
+description = ""
+authors = ["Burak Kadir Er <ksenofanex@gmail.com>", "MilaDog <daniel.ryan.sergeant@gmail.com>", "Mmesek"]
+license = "MIT"
+
+[tool.poetry.dependencies]
+python = "^3.10"
```

Comment:
Actually, it's a code jam project, wouldn't it be better to have version pinned, like in [sample](https://github.com/python-discord/code-jam-template/blob/1ced0aa734ecca6744f9ed078a4eab72c3d09058/samples/pyproject.toml#L9) file rather than allowing later versions? Template [mentions](https://github.com/python-discord/code-jam-template/tree/main#how-do-i-adapt-it-to-my-project) pinning dependencies so the project doesn't break after (major) change in API when jam ends

Reply:
Nice catch.